pandas/tests/groupby/test_nth.py

```diff
@@ -726,6 +726,15 @@ def test_slice(slice_test_df, slice_test_grouped, arg, expected_rows):
     tm.assert_frame_equal(result, expected)
 
 
+def test_nth_indexed(slice_test_df, slice_test_grouped):
```

Comment:
worth testing some more cases here,? e.g. and can compre vs a `slice()` object?

Reply:
Added indexed equivalent to test_slices() and call equivalent to test_nth_indexed(). 
The literal index cannot really be parameterized, but I think that these tests are enough to establish equivalence.